<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab{
background:#2054ff;
color:white;
}
QTabBar::tab:selected{
background:rgb(170, 0, 0);
color:white;
}
QTabWidget::pane { /* The tab widget frame */
border-top: 0px solid #EA9D9D;
}
QTabWidget::tab-bar {
left: 5px; /* move to the right by 5px */
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>materiel</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab{
background:#2054ff;
color:white;
}
QTabBar::tab:selected{
background:rgb(170, 0, 0);
color:white;
}
QTabWidget::pane { /* The tab widget frame */
border-top: 0px solid #EA9D9D;
}
QTabWidget::tab-bar {
left: 5px; /* move to the right by 5px */
}</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>matricule</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>model</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}
</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>160</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(0, 0, 255);
color:black;
}</string>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>etat</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}
QComboBox::down-arrow {

}</string>
        </property>
        <item>
         <property name="text">
          <string>marche</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>en panne</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>120</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}
</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>groupe</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>641</width>
          <height>301</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {
    gridline-color: white;
color:white;
   background-color:rgba(0,0,0,50%);
border:0px;
border-radius: 5px;
    }
QHeaderView {background-color: transparent;
color:white;

}
QHeaderView::section{
border-style: none;
background-color: transparent;
margin-bottom:5px;
margin-top:5px;
border:0px;
}
QTableView QTableCornerButton::section {
     bottom-style:none;
border:0px;

 }
</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>340</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(0, 0, 255);
color:black;
}</string>
        </property>
        <property name="text">
         <string>pdf </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>matricule*</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>170</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(0, 0, 255);
color:black;
}</string>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>414</x>
          <y>130</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>etat</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_20">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>361</width>
          <height>231</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {
    gridline-color: white;
color:white;
   background-color:rgba(0,0,0,50%);
border:0px;
border-radius: 5px;
    }
QHeaderView {background-color: transparent;
color:white;

}
QHeaderView::section{
border-style: none;
background-color: transparent;
margin-bottom:5px;
margin-top:5px;
border:0px;
}
QTableView QTableCornerButton::section {
     bottom-style:none;
border:0px;

 }
</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_3">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>130</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}
QComboBox::down-arrow {

}</string>
        </property>
        <item>
         <property name="text">
          <string>marche</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>en panne</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>supprimer</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_12">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(0, 0, 255);
color:black;
}</string>
        </property>
        <property name="text">
         <string>suprimer</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>671</width>
          <height>192</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {
    gridline-color: white;
color:white;
   background-color:rgba(0,0,0,50%);
border:0px;
border-radius: 5px;
    }
QHeaderView {background-color: transparent;
color:white;

}
QHeaderView::section{
border-style: none;
background-color: transparent;
margin-bottom:5px;
margin-top:5px;
border:0px;
}
QTableView QTableCornerButton::section {
     bottom-style:none;
border:0px;

 }
</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>matricule</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>travaille</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab{
background:#2054ff;
color:white;
}
QTabBar::tab:selected{
background:rgb(170, 0, 0);
color:white;
}
QTabWidget::pane { /* The tab widget frame */
border-top: 0px solid #EA9D9D;
}
QTabWidget::tab-bar {
left: 5px; /* move to the right by 5px */
}</string>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID policier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>nom du groupe</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>nombre </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>description</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}
</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>180</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(0, 0, 255);
color:black;
}</string>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>641</width>
          <height>301</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {
    gridline-color: white;
color:white;
   background-color:rgba(0,0,0,50%);
border:0px;
border-radius: 5px;
    }
QHeaderView {background-color: transparent;
color:white;

}
QHeaderView::section{
border-style: none;
background-color: transparent;
margin-bottom:5px;
margin-top:5px;
border:0px;
}
QTableView QTableCornerButton::section {
     bottom-style:none;
border:0px;

 }
</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>340</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(0, 0, 255);
color:black;
}</string>
        </property>
        <property name="text">
         <string>pdf</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_15">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>130</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(0, 0, 255);
color:black;
}</string>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>414</x>
          <y>130</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>description</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_17">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}
</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>361</width>
          <height>231</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {
    gridline-color: white;
color:white;
   background-color:rgba(0,0,0,50%);
border:0px;
border-radius: 5px;
    }
QHeaderView {background-color: transparent;
color:white;

}
QHeaderView::section{
border-style: none;
background-color: transparent;
margin-bottom:5px;
margin-top:5px;
border:0px;
}
QTableView QTableCornerButton::section {
     bottom-style:none;
border:0px;

 }
</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID policier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>supprimer</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color:rgba(155,170,255,20%);
border:1 solid blue;
border-radius:5px;
color:black;
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgba(0,0,0,50%);
color:white;
border:1 solid white;
border-radius: 5px;
}
QPushButton:hover{
background-color:rgb(0, 0, 255);
color:black;
}</string>
        </property>
        <property name="text">
         <string>suprimer</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>671</width>
          <height>192</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView {
    gridline-color: white;
color:white;
   background-color:rgba(0,0,0,50%);
border:0px;
border-radius: 5px;
    }
QHeaderView {background-color: transparent;
color:white;

}
QHeaderView::section{
border-style: none;
background-color: transparent;
margin-bottom:5px;
margin-top:5px;
border:0px;
}
QTableView QTableCornerButton::section {
     bottom-style:none;
border:0px;

 }
</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID policier</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1243</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
